client/src/api/BackendAccess.tsx

```diff
@@ -1,16 +1,151 @@
 
-const API_URL = process.env.REACT_APP_APIBASEURL as string;
+const apiBaseUrl = process.env.REACT_APP_APIBASEURL as string;
 
-export async function loadChores() {
-  const apiAccessToken = ""; // useRecoilValue(AppState.userBearerToken); TODO not implemented as hook
+// GET
+export function QueryGet(key: string, url: queryUrl): ChoreLoadingData {
+  const bearerToken = useRecoilValue(AppState.userBearerToken);
+  const [queryState, setqueryState] = useRecoilState(AppState.queryStateInfo);
+  const showError = useErrorNotification();
 
-  //Todo: Implement axios with async await...
-  return axios.get(API_URL + "/Chores", {
-    headers: {
-      Authorization: `Bearer ${apiAccessToken}`,
-    },
-  });
+  const { isLoading, isFetching, isError, error, data } = useQuery(key, () =>
+    axios
+      .get(`${apiBaseUrl}${url}`, {
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+      })
+      .then((data) => data?.data)
+  );
+  if ((isLoading || isFetching) && queryState.open === false) {
+    setqueryState({
+      open: true,
+    });
+  } else if (isError) {
+    console.log(error);
+    showError("losed Data!");
+  } else if (
+    !isLoading &&
+    !isFetching &&
+    !isError &&
+    queryState.open === true
+  ) {
+    setqueryState({
+      open: false,
+    });
+  }
+  return {
+    isLoading: isLoading,
+    isFetching: isFetching,
+    isError: isError,
+    error: error,
+    data: data ?? [],
+  };
+}
+
+// POST
+export function QueryPost(): UseMutationResult<
+  AxiosResponse<QueryObject>,
+  unknown,
+  QueryObject,
+  unknown
+> {
+  const bearerToken = useRecoilValue(AppState.userBearerToken);
+  const queryClient = useQueryClient();
+  const showSuccess = useSuccessNotification();
+  const mutation = useMutation(
+    (object: QueryObject) =>
+      axios.post<QueryObject>(`${apiBaseUrl}${object.url}`, object.object, {
+        headers: {
+          Authorization: `Bearer ${bearerToken}`,
+        },
+      }),
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries("parentChoreGet");
+        showSuccess("chore created");
+      },
+    }
+  );
+
+  return mutation;
+}
+
+// PUT
+export function QueryPut(): UseMutationResult<
+  AxiosResponse<QueryObject>,
+  unknown,
+  QueryObject,
+  unknown
+> {
+  const bearerToken = useRecoilValue(AppState.userBearerToken);
+  const queryClient = useQueryClient();
+  const showInfo = useInfoNotification();
+  const mutation = useMutation(
+    (object: QueryObject) =>
+      axios.put<QueryObject>(
+        `${apiBaseUrl}${object.url + object.object.id}`,
+        object.object,
+        {
+          headers: {
+            Authorization: `Bearer ${bearerToken}`,
+          },
+        }
+      ),
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries("parentChoreGet");
+      },
+    }
+  );
+
+  return mutation;
+}
+
+// DELETE
+export function QueryDelete(): UseMutationResult<
```

Comment:
Die Hooks, wie schon erwähnt, bitte mit "use" präfixen.
Vielleicht könnte man den Namen auch verkürzen in useGet, usePost, usePut, useDelete?
Die Verwendung von Generics noch umsetzen.